ambry-router/src/main/java/com.github.ambry.router/RouterUtils.java

```diff
@@ -76,6 +76,7 @@ static boolean isSystemHealthError(Exception exception) {
         case BadInputChannel:
         case BlobDeleted:
         case BlobExpired:
+        case BlobDoesNotExist:
```

Comment:
not related to this change, but if i'm adding a new error code (which is in API), how do i know i have to come add something in an implementation? Seems like a bad dependency.